applications/MeshingApplication/python_scripts/mmg_process.py

```diff
@@ -122,10 +121,10 @@ def __init__(self,Model,params):
                 mean = stat.mean(nodal_h_values)
                 stdev = stat.stdev(nodal_h_values)
                 prob = (self.params["automatic_remesh_parameters"]["min_size_current_percentage"].GetDouble())/100
-                self.params["minimal_size"].SetDouble(normvalf(prob, mean, stdev))
+                self.params["minimal_size"].SetDouble(_normvalf(prob, mean, stdev))
```

Comment:
what are u doing here? just put a comment so that it is understandable on reading

Reply:
I use a normal distribution to get the minimal size as  a stadistical meaninful value